### ISSUE TITLE
error: Unsupported type: Build.LazyPath @ Zig 0.13.0

### ISSUE THREAD

**Issue by ElrohirGT** [karlseguin/pg.zig#54]
Hi! I'm kinda new to zig and wanted to try connecting to a postgres DB. I searched on the internet and your library seems pretty awesome! Tried to install it like the README said but when I try to compile my script it fails with:

```
[elrohirgt@elrohirgt:~/Documents/Development/JUWURA/backend]$ zig build -freference-trace
/nix/store/j9sp79vdhz4l6lfnvzhvcprhar803zr3-zig-0.13.0/lib/zig/std/Build.zig:1532:21: error: Unsupported type: Build.LazyPath
            else => @compileError("Unsupported type: " ++ @typeName(T)),
                    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/nix/store/j9sp79vdhz4l6lfnvzhvcprhar803zr3-zig-0.13.0/lib/zig/std/Build.zig:1087:40: note: called from here
    const type_id = comptime typeToEnum(T);
                             ~~~~~~~~~~^~~
referenced by:
    build: /home/elrohirgt/.cache/zig/p/1220fb3f43073a6f9f201764c3ae78aa87b29edbc7b021c1174b29b020967d0b146c/build.zig:31:42
    runBuild__anon_16318: /nix/store/j9sp79vdhz4l6lfnvzhvcprhar803zr3-zig-0.13.0/lib/zig/std/Build.zig:2117:37
    dependencyInner__anon_15177: /nix/store/j9sp79vdhz4l6lfnvzhvcprhar803zr3-zig-0.13.0/lib/zig/std/Build.zig:2097:29
    dependency__anon_14239: /nix/store/j9sp79vdhz4l6lfnvzhvcprhar803zr3-zig-0.13.0/lib/zig/std/Build.zig:1954:35
```

Since the only thing I changed before compilation was adding the library to the build.zig.zon and build.zig I tried to clone your repo and build it too. I fails with more or less the same error:

```
[elrohirgt@elrohirgt:~/Documents/Development/pg.zig]$ zig build -freference-trace
/nix/store/j9sp79vdhz4l6lfnvzhvcprhar803zr3-zig-0.13.0/lib/zig/std/Build.zig:1532:21: error: Unsupported type: Build.LazyPath
            else => @compileError("Unsupported type: " ++ @typeName(T)),
                    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/nix/store/j9sp79vdhz4l6lfnvzhvcprhar803zr3-zig-0.13.0/lib/zig/std/Build.zig:1087:40: note: called from here
    const type_id = comptime typeToEnum(T);
                             ~~~~~~~~~~^~~
referenced by:
    build: /home/elrohirgt/Documents/Development/pg.zig/build.zig:31:42
    runBuild__anon_8821: /nix/store/j9sp79vdhz4l6lfnvzhvcprhar803zr3-zig-0.13.0/lib/zig/std/Build.zig:2117:37
    main: /nix/store/j9sp79vdhz4l6lfnvzhvcprhar803zr3-zig-0.13.0/lib/zig/compiler/build_runner.zig:301:29
    callMain: /nix/store/j9sp79vdhz4l6lfnvzhvcprhar803zr3-zig-0.13.0/lib/zig/std/start.zig:524:32
    callMainWithArgs: /nix/store/j9sp79vdhz4l6lfnvzhvcprhar803zr3-zig-0.13.0/lib/zig/std/start.zig:482:12
    posixCallMainAndExit: /nix/store/j9sp79vdhz4l6lfnvzhvcprhar803zr3-zig-0.13.0/lib/zig/std/start.zig:438:20
    _start: /nix/store/j9sp79vdhz4l6lfnvzhvcprhar803zr3-zig-0.13.0/lib/zig/std/start.zig:351:40
```

Whats really weird is the line 31 and 30 are practically the same but if only fails on 31. Am I using the incorrect version of Zig? Do I need to configure SSL even if I'm not going to use it?

**Comment by ElrohirGT** [comment]
I realized I didn't really supply my build.zig script, here's the relevant part:

```zig
const pg = b.dependency("pg", .{
        .target = target,
        .optimize = optimize,
    });
    main_exe.root_module.addImport("pg", pg.module("pg"));
```

**Comment by 4lxprime** [comment]
I had the exact same issue and it worked with the latest version of zig (`0.14.0`, master, https://ziglang.org/builds/zig-linux-x86_64-0.14.0-dev.2384+cbc05e0b1.tar.xz).

However this version is not a stable release and must not be the only one working, maybe it could be fixed with a different build system depending on the zig version (with `builtin.zig_version`).

**Comment by ElrohirGT** [comment]
I'm so dumb, I didn't realize there were different branches according to the Zig version. For anyone wondering I fixed deleting the lines on my build.zig.zon that referenced this library and executed the following command:

```bash
zig fetch --save git+https://github.com/karlseguin/pg.zig#zig-0.13
```

Now it compiles and works as expected! I can also checkout the example in the branch for 0.13. Closing this since it was a skill issue.

Maybe have an announcement in the README to install the same version would be an improvement?
